api/institutions/views.py

```diff
@@ -435,38 +424,35 @@ def _make_elasticsearch_results_filterable(self, departments: dict) -> MockQuery
 
         queryset = MockQueryset()
 
-        institution_id = self.get_institution()._id
-        for department in departments:
-            department.update({'id': institution_id})
-            queryset.add_dict_as_item(department)
+        for item in items:
+            item.update(kwargs)
+            queryset.add_dict_as_item(item)
```

Comment:
Similarly, elastic can do filtering. Why don't we take advantage of that?

Reply:
Yes! and there's a test for it in a the department metrics file.